src/HttpClient/VerboseErrorHttpClient.php

```diff
@@ -27,9 +28,16 @@ public function request(string $method, string $url, array $options = []): Respo
         return new VerboseErrorResponse($response);
     }
 
-    public function stream($responses, ?float $timeout = null): ResponseStreamInterface
+    /**
+     * @param VerboseErrorResponse|iterable $responses
```

Comment:
Kan de iterable ook niet typ-hinted zijn?
```suggestion
     * @param VerboseErrorResponse|iterable<VerboseErrorResponse> $responses
```